lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -676,3 +676,132 @@ end
 e2function entity lastDisconnectedPlayer()
 	return lastLeft
 end
+
+----- Deaths+Spawns, Dev: Vurv, 12/10/19 -----
+local DeathAlert = {} -- table of e2s that have runOnDeath(1)
+local RespawnAlert = {}
+local DeathList = { last = {NULL, NULL, NULL, 0} }
+local RespawnList = { last = {NULL,0} }
+local function DEFAULT_TABLE() return {n={},ntypes={},s={},stypes={},size=0} end
+
+hook.Add("PlayerDeath","Exp2PlayerDetDead",function(victim,inflictor,attacker)
+	local entry = DEFAULT_TABLE() -- default table
+	entry.s["Victim"]=victim
+	entry.s["Inflictor"]=inflictor
+	entry.s["Attacker"]=attacker
+	entry.s["Timestamp"]=CurTime()
+	entry.stypes["Victim"]="e"
+	entry.stypes["Inflictor"]="e"
+	entry.stypes["Attacker"]="e"
+	entry.stypes["Timestamp"]="n"
+	entry.n[1]=victim
+	entry.n[2]=inflictor
+	entry.n[3]=attacker
+	entry.n[4]=CurTime()
+	entry.ntypes[1]="e"
+	entry.ntypes[2]="e"
+	entry.ntypes[3]="e"
+	entry.ntypes[4]="n"
+	entry.size=4
```

Comment:
The size should be 8

Reply:
There's 4 objects in it though

Reply:
["Victim"]
["Inflictor"]
["Attacker"]
["Timestamp"]
[1]
[2]
[3]
[4]

that's 8

Reply:
Ah all right, makes sense.

Reply:
@Vurv78 alternatively, you can leave the n and ntypes tables empty.